bricks/permission_handler/brick.yaml

```diff
@@ -0,0 +1,28 @@
+name: permission_handler
+description: A new brick created with the Mason CLI.
+
+# The following defines the version and build number for your brick.
+# A version number is three numbers separated by dots, like 1.2.34
+# followed by an optional build number (separated by a +).
+version: 0.1.0+1
+
+# The following defines the environment for the current brick.
+# It includes the version of mason that the brick requires.
+environment:
+  mason: ">=0.1.0-dev.41 <0.1.0"
+
+# Variables specify dynamic values that your brick depends on.
+# Zero or more variables can be specified for a given brick.
+# Each variable has:
+#  * a type (string, number, boolean, enum, or array)
+#  * an optional short description
+#  * an optional default value
+#  * an optional list of default values (array only)
+#  * an optional prompt phrase used when asking for the variable
+#  * a list of values (enums only)
+vars:
+  project_name:
+    type: string
+    description: Your project name
+    default: your_project_name
+    prompt: Project name?
```

Comment:
Ah, it will be injected from the root brick. 
- Should we add a comment here?
- As only `type` is mandatory. Could we get rid of the redundant fields?

```suggestion
vars:
  # Injected from the root brick by post_gen hook
  project_name:
    type: string
```

Reply:
@luongvo removed in [b41d394](https://github.com/nimblehq/flutter-templates/pull/203/commits/b41d39478c5a95ec11cc7f09f6966bf6a814f548)